mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifierOptions.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "mediapipe/tasks/ios/components/processors/sources/MPPClassifierOptions.h"
+#import "mediapipe/tasks/ios/core/sources/MPPTaskOptions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Options for setting up a `MPPTextClassifierOptions`.
+ */
+NS_SWIFT_NAME(TextClassifierOptions)
+@interface MPPTextClassifierOptions : MPPTaskOptions
```

Comment:
@schmidt-sebastian @khanhlvg  Instead of inling the classifier options in each task like the[ java](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/java/com/google/mediapipe/tasks/text/textclassifier/TextClassifier.java#L211) implementation, I have followed the [C++ ](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/cc/text/text_classifier/text_classifier.h#L39) approach of having a `MPPClassifierOptions` class  which can be used by all classification tasks like AudioClassifier, TextClassifier, ImageClassifier etc.
LMK if the Java approach needs to be followed.